scripts/test.sh

```diff
@@ -14,5 +14,6 @@ cd "$(dirname $0)"
 ../contracts/near/context-proxy/build-test-deps.sh
 ../contracts/icp/context-config/build.sh
 ../contracts/icp/context-proxy/build_contracts.sh
+../contracts/stellar/context-proxy/build_all_contracts.sh
```

Comment:
let's follow the existing pattern, independently build context-config, instead of asking context-proxy to do it